maintainers/maintainer-list.nix

```diff
@@ -7,17 +7,32 @@
 
       # Optional
       github = "GithubUsername";
+      pgpLongId = "0x0123456789ABCDEF";
+      pgpFingerprint = "AAAA BBBB CCCC DDDD EEEE  FFFF 0000 1111 2222 3333";
     };
 
   where
 
   - `handle` is the handle you are going to use in nixpkgs expressions,
   - `name` is your, preferably real, name,
   - `email` is your maintainer email address, and
-  - `github` is your GitHub handle (as it appears in the URL of your profile page, `https://github.com/<userhandle>`).
+  - `github` is your GitHub handle (as it appears in the URL of your profile page, `https://github.com/<userhandle>`),
+  - `pgpLongId` is your long-form PGP/GPG keyID,
+  - `pgpFingerprint` is your PGP/GPG key fingerprint.
 
   `handle == github` is strongly preffered whenever `github` is an acceptable attribute name and is short and convenient.
 
+  Only add PGP/GPG values if you actually use them to sign commits and/or mail.
+
+  To get the required PGP/GPG values, add the following to your `~/.gnupg/gpg.conf`:
+  ```
+  keyid-format 0xlong
+  with-fingerprint
+  ```
+  and then run `gpg -k`.
+
+  !!! Note that PGP/GPG values stored here are for informational purposes only, don't use this file as a singe source of truth.
```

Comment:
Is there a way to check keys against the github API?

Reply:
There is `https://api.github.com/users/tilpner/gpg_keys` in the v3 API. This might indeed be used for the initial population of this list